src/mail.c

```diff
@@ -0,0 +1,739 @@
+    	.linesLayout = sLayout_3x3
+    },
+};
+
+static const struct MailEcWordLayout sLayout_5x2[] = {
+    { .numWordsInLine = 2, .lineXoffset = 0, .lineHeight = 16 },
+    { .numWordsInLine = 2, .lineXoffset = 0, .lineHeight = 16 },
+    { .numWordsInLine = 2, .lineXoffset = 0, .lineHeight = 16 },
+    { .numWordsInLine = 2, .lineXoffset = 0, .lineHeight = 16 },
+    { .numWordsInLine = 1, .lineXoffset = 0, .lineHeight = 16 }
+};
+
+static const struct MailAttrStruct sMessageLayouts_5x2[] = {
+    [ITEM_TO_MAIL(ITEM_ORANGE_MAIL)]  = {
+      	.numRows = 5,
+      	.nameY = 0,
+      	.nameX = 8,
+      	.messageTop = 3,
+      	.messageLeft = 8,
+      	.linesLayout = sLayout_5x2
+    },
+    [ITEM_TO_MAIL(ITEM_HARBOR_MAIL)]  = {
+      	.numRows = 5,
+      	.nameY = 3,
+      	.nameX = 14,
+      	.messageTop = 3,
+      	.messageLeft = 8,
+      	.linesLayout = sLayout_5x2
+    },
+    [ITEM_TO_MAIL(ITEM_GLITTER_MAIL)] = {
+      	.numRows = 5,
+      	.nameY = 0,
+      	.nameX = 16,
+      	.messageTop = 3,
+      	.messageLeft = 8,
+      	.linesLayout = sLayout_5x2
+    },
+    [ITEM_TO_MAIL(ITEM_MECH_MAIL)]    = {
+      	.numRows = 5,
+      	.nameY = 0,
+      	.nameX = 14,
+      	.messageTop = 3,
+      	.messageLeft = 8,
+      	.linesLayout = sLayout_5x2
+    },
+    [ITEM_TO_MAIL(ITEM_WOOD_MAIL)]    = {
+      	.numRows = 5,
+      	.nameY = 3,
+      	.nameX = 12,
+      	.messageTop = 3,
+      	.messageLeft = 8,
+      	.linesLayout = sLayout_5x2
+    },
+    [ITEM_TO_MAIL(ITEM_WAVE_MAIL)]    = {
+      	.numRows = 5,
+      	.nameY = 0,
+      	.nameX = 18,
+      	.messageTop = 3,
+      	.messageLeft = 8,
+      	.linesLayout = sLayout_5x2
+    },
+    [ITEM_TO_MAIL(ITEM_BEAD_MAIL)]    = {
+      	.numRows = 5,
+      	.nameY = 0,
+      	.nameX = 20,
+      	.messageTop = 3,
+      	.messageLeft = 8,
+      	.linesLayout = sLayout_5x2
+    },
+    [ITEM_TO_MAIL(ITEM_SHADOW_MAIL)]  = {
+      	.numRows = 5,
+      	.nameY = 6,
+      	.nameX = 20,
+      	.messageTop = 3,
+      	.messageLeft = 8,
+      	.linesLayout = sLayout_5x2
+    },
+    [ITEM_TO_MAIL(ITEM_TROPIC_MAIL)]  = {
+      	.numRows = 5,
+      	.nameY = 0,
+      	.nameX = 16,
+      	.messageTop = 3,
+      	.messageLeft = 8,
+      	.linesLayout = sLayout_5x2
+    },
+    [ITEM_TO_MAIL(ITEM_DREAM_MAIL)]   = {
+      	.numRows = 5,
+      	.nameY = 0,
+      	.nameX = 14,
+      	.messageTop = 3,
+      	.messageLeft = 8,
+      	.linesLayout = sLayout_5x2
+    },
+    [ITEM_TO_MAIL(ITEM_FAB_MAIL)]     = {
+      	.numRows = 5,
+      	.nameY = 8,
+      	.nameX = 16,
+      	.messageTop = 3,
+      	.messageLeft = 8,
+      	.linesLayout = sLayout_5x2
+    },
+    [ITEM_TO_MAIL(ITEM_RETRO_MAIL)]   = {
+      	.numRows = 5,
+      	.nameY = 0,
+      	.nameX = 16,
+      	.messageTop = 3,
+      	.messageLeft = 8,
+      	.linesLayout = sLayout_5x2
+    },
+};
+
+void ReadMail(struct MailStruct * mail, void (* savedCallback)(void), bool8 messageExists)
```

Comment:
inconsistent func pointer space style with line 63 & 64. But this style in the header :/